src/main/java/org/isf/anamnesis/model/PatientHistory.java

```diff
@@ -218,7 +218,7 @@ public class PatientHistory extends Auditable<String> implements Comparable<Pati
 	private int phyMenopauseYears;
 
 	@Column(name = "PAH_PHY_HRT_NOR")
-	private boolean phyHrtNormal;
+	private boolean phyHrtNormal = true;
```

Comment:
it's ok... should we follow same "default 1" values for all fields like in the step_82 script? (I didn't check if we do)
Also, is there a way to do it by annotation instead of by variable assignation? (just wondering what is the best practise)

Reply:
I confirm that it is not following the default values like in the step_82 script. Should I follow the values of UI or those of the script_82? There is a way to do it with annotations, like @PrePersist, but it is cumbersome, there are to many fields and we should use wrappers not primitive types. @ColumnDefault instead is usefull when you generate the DB with hibernate, but we don't do that.

Reply:
Ok, let's stay this way, thanks!